src/desktop-lite/install.sh

```diff
@@ -377,7 +384,7 @@ screen_geometry="\${VNC_RESOLUTION%*x*}"
 screen_depth="\${VNC_RESOLUTION##*x}"
 
 # Check if VNC_PASSWORD is set and use the appropriate command
-common_options="tigervncserver \${DISPLAY} -geometry \${screen_geometry} -depth \${screen_depth} -rfbport ${VNC_PORT} -dpi \${VNC_DPI:-96} -localhost -desktop fluxbox -fg"
```

Comment:
Shouldn't we conditionally add `-desktop fluxbox` based upon `INSTALL_FLUXBOX`?

Reply:
@samruddhikhandale At first I thought so but this doesn't matter, `-desktop` is only used to give a name to the session that the client sees.

By default, the value for this is "x11", but it can be anything and has no impact whatsoever, so I don't think that it's needed.